apps/app/src/views/components/Auth.tsx

```diff
@@ -30,7 +30,10 @@ export const Auth = (props: AuthProps) => {
     );
 
   return (
-    <Button colorScheme="blue" onClick={() => login({ appState: { returnTo } })}>
+    <Button
+      colorScheme="blue"
+      onClick={() => login({ appState: { returnTo: window.location.origin + returnTo } })}
```

Comment:
can you make this a template literal? 
Does it need the origin?

Reply:
I removed the origin and it still works ty